test/new-e2e/runner/ci_profile.go

```diff
@@ -13,9 +13,10 @@ import (
 	"github.com/DataDog/datadog-agent/test/new-e2e/runner/parameters"
 )
 
-const defaultCISecretPrefix = "ci.datadog-agent."
-
-var defaultCIEnvs = []string{"aws/agent-qa"}
+const (
+	defaultCISecretPrefix = "ci.datadog-agent."
+	defaultCIEnv          = "aws/agent-qa"
```

Comment:
```suggestion
	defaultCIEnvironments          = "aws/agent-qa"
```

Reply:
Another developer might say that `defaultCIEnvironments` is a string, thus a single environment.

Reply:
Will commit, no strong opinion